tests/test-backend-ops.cpp

```diff
@@ -3912,7 +3912,7 @@ static const ggml_type other_types[] = {
 // Test cases for evaluation: should try to cover edge cases while using small input sizes to keep the runtime low
 static std::vector<std::unique_ptr<test_case>> make_test_cases_eval() {
     std::vector<std::unique_ptr<test_case>> test_cases;
-    std::default_random_engine rng(0);
+    [[maybe_unused]] std::default_random_engine rng(0);
```

Comment:
I'm not confident this can be used with all the compilers supported in llama or would be accepted in general. It's best to move it in a separate PR if you're keen to fix this IMO.

Reply:
This will fail e.g. for gcc 7.5 ...